src/python/pants/option/global_options.py

```diff
@@ -215,26 +251,21 @@ def register_bootstrap_options(cls, register):
             default=[],
             advanced=True,
             help="Regexps matching warning strings to ignore, e.g. "
-            '["DEPRECATED: scope some_scope will be removed"]. The regexps will be matched '
-            "from the start of the warning string, and will always be case-insensitive. "
-            "See the `warnings` Python module documentation for more background "
-            "on how these are used.",
```

Comment:
This seemed like a leak of the implementation.

Reply:
Possibly we *want* to leak the implementation here, so users can know how to get their regexes to work.